packages/components/src/MultiSelect/_docs/MultiSelect.stories.tsx

```diff
@@ -3,7 +3,7 @@ import { Meta, StoryObj } from "@storybook/react"
 import { MultiSelect, MultiSelectProps } from "../index"
 
 const meta = {
-  title: "Staging/Multi Select",
+  title: "Components/Multi Select",
```

Comment:
This will be the new convention (not sure where it was documented) - we can fix it separately if there's nothing else to fix though.

```suggestion
  title: "Components/MultiSelect",
```